website/src/app/page.tsx

```diff
@@ -370,7 +369,7 @@ export default function Page() {
                 </ChromeIcon>
               </div>
               <div className="p-4 flex items-center justify-center bg-white rounded-lg border border-2 border-neutral-200">
-                <AppleIcon size={12} href="/kb/user-guides/macos-client">
+                <AppleIcon size={12} href="/kb/user-guides/ios-client">
```

Comment:
I guess the Apple Clients used to share one page and then they split into macOS and iOS? the iOS link was still pointed to macOS so I fixed that.

Reply:
They both still say "Apple Clients" though